warehouse/templates/pages/sponsor.html

```diff
@@ -0,0 +1,244 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "base.html" %}
+{% block title %}{% trans %}Sponsor the Python Packaging Working Group{% endtrans %}{% endblock %}
+
+{% block content %}
+<div class="horizontal-section">
+  <div class="site-container">
+    <div class="sponsorship-intro">
+      <div class="sponsorship-intro__text">
+        <div class="heading-wsubtitle">
+          <h1 class="heading-wsubtitle__heading">{% trans %}Sponsor PyPI and related projects{% endtrans %}</h1>
+          <p class="heading-wsubtitle__subtitle">{% trans %}Donate to the Python Packaging Working Group{% endtrans %}</p>
+        </div>
+        <p class="lede-paragraph">
+          {% trans href='https://wiki.python.org/psf/PackagingWG', title=gettext('External link') %}The <a href="{{ href }}" title="{{ title }}" target="_blank" rel="noopener">Python Packaging Working Group</a> is a work group of the Python Software Foundation which raises and distributes funds to improve Python's packaging ecosystem.{% endtrans %}
+        </p>
+        <p>{% trans %}Recent projects funded through the working group include:{% endtrans %}</p>
+        <ul class="hooray-list">
+          <li>{% trans %}The successful relaunch of the Python Package Index, powered by the new 'Warehouse' codebase{% endtrans %}</li>
```

Comment:
Would it at all make any sense to put this list in reverse order, and put a limit on it and start dropping items once it gets more than N entries?

Reply:
I think the most visible project for the community has been the warehouse launch, so that's why I put it first...
Not sure we need to limit it - maybe a better solution would be to schedule an annual review on this entire page, to update this and any other information?